tests/api/templates.test.js

```diff
@@ -105,4 +112,76 @@ describe("Test JSON validation scenarios", () => {
     expect(res.statusCode).toBe(400);
     expect(JSON.parse(res._getData()).error).toContain('instance requires property "form"');
   });
+
+  it("Should log admin activity if POST API call completed successfully", async () => {
+    const mockSession = {
+      expires: "1",
+      user: { email: "forms@cds.ca", name: "forms", admin: true, id: 1 },
+    };
+    client.getSession.mockReturnValue(mockSession);
+
+    const { req, res } = createMocks({
+      method: "POST",
+      headers: {
+        "Content-Type": "application/json",
+        Origin: "http://localhost:3000",
+      },
+      body: {
+        formConfig: validFormTemplate,
+      },
+    });
+
+    await templates(req, res);
+
+    expect(res.statusCode).toBe(200);
+    expect(logFormUpload).toHaveBeenCalled();
+  });
+
+  it("Should log admin activity if PUT API call completed successfully", async () => {
```

Comment:
Minor nit but this test is not only testing logging admin activity. Perhaps it would be better suited to rename to `Should successfully handle PUT request`